Ginger/Ginger/UserControlsLib/UCSendEMailConfig/UCSendEMailConfigView.xaml

```diff
@@ -0,0 +1,511 @@
+                        </Grid>
+                        <Grid Margin="{StaticResource RowMargin}">
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="{StaticResource LabelColumnWidth}"/>
+                                <ColumnDefinition Width="{StaticResource ValueColumnWidth}"/>
+                            </Grid.ColumnDefinitions>
+                            <Label Grid.Column="0" Style="{StaticResource @InputFieldLabelStyle}" VerticalAlignment="Center" HorizontalAlignment="Left" Content="Comment:"/>
+                            <Actions:UCValueExpression x:Name="xBodyCommentVE" Grid.Column="1"/>
+                        </Grid>
+                    </StackPanel>
+                </StackPanel>
+            </TabItem>
+            <TabItem x:Name="xAttachmentsTab" Background="{StaticResource $BackgroundColor_LightGray}" Visibility="Collapsed" >
+                <TabItem.Header>
+                    <Label Content="Attachments"/>
+                </TabItem.Header>
+                <TabItem.Resources>
+                    <DataTemplate x:Key="ZippedCellTemplate">
+                        <CheckBox x:Name="xAttachmentZippedCheckBox" Style="{StaticResource @CheckBoxStyle}" VerticalAlignment="Top" HorizontalAlignment="Center"/>
+                    </DataTemplate>
+                    <DataTemplate x:Key="NameVEButtonCellTemplate" >
+                        <Button x:Name="xAttachmentNameVEButton" Content="..." Click="xAttachmentNameVEButton_Click"  Style="{DynamicResource DataGridCellButtonStyle}"/>
+                    </DataTemplate>
+                </TabItem.Resources>
+                <Ginger:ucGrid x:Name="xAttachmentsGrid" Title="Attachments" ShowAdd="Collapsed" ShowEdit="Collapsed" ShowRefresh="Collapsed" ShowUpDown="Collapsed"/>
+            </TabItem>
+            <TabItem x:Name="xDeliveryTab" Background="{StaticResource $BackgroundColor_LightGray}">
+                <TabItem.Header>
+                    <Label Content="Configuration"/>
+                </TabItem.Header>
+                <StackPanel Orientation="Vertical" Background="{StaticResource $BackgroundColor_LightGray}">
+                    <Grid Margin="{StaticResource RowMargin}">
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="{StaticResource LabelColumnWidth}"/>
+                            <ColumnDefinition Width="{StaticResource ValueColumnWidth}"/>
+                        </Grid.ColumnDefinitions>
+                        <Label Grid.Column="0" Grid.Row="1" Style="{StaticResource @InputFieldLabelStyle}" VerticalAlignment="Center" HorizontalAlignment="Left" Content="Email Method:"/>
+                        <ComboBox x:Name="xEmailMethod" SelectionChanged="xEmailMethod_SelectionChanged" Grid.Column="1" Style="{StaticResource @InputComboBoxStyle}">
+                            <ComboBoxItem x:Name="xEmailMethodSMTP" Content="SMTP" Tag="SMTP"/>
```

Comment:
Dropdown is not having MS Graph API option. is it not the same UC?

Reply:
As we discussed, since this change will require a lot of modifications and thorough testing, we will skip this for now